### PR TITLE
Update terminology client to use url param

### PR DESCRIPTION
**Pull request by TahaAttari**

### CODE REVIEW

File: cqf-fhir-utility/src/test/java/org/opencds/cqf/fhir/utility/client/TerminologyServerClientTest.java [hunk 1]
```diff
@@ -54,13 +54,27 @@ void testR4UrlAndVersion() {
 
         // when the valueset has a version it should be in params
         vs.setVersion(version);
-        client.expand(vs, authoritativeSource, new org.hl7.fhir.r4.model.Parameters(), username, password);
+        client.expand(vs, authoritativeSource, null, username, password);
```
Comment (TahaAttari) [inline cqf-fhir-utility/src/test/java/org/opencds/cqf/fhir/utility/client/TerminologyServerClientTest.java:57]: these changes are to add a bit more coverage

File: cqf-fhir-utility/src/main/java/org/opencds/cqf/fhir/utility/client/TerminologyServerClient.java [hunk 2]
```diff
@@ -13,6 +13,54 @@ public TerminologyServerClient(FhirContext ctx) {
...
+            org.hl7.fhir.r4.model.Parameters expansionParameters,
+            String username,
+            String apiKey,
+            String url,
+            String version) {
+        if (expansionParameters == null) {
+            expansionParameters = new org.hl7.fhir.r4.model.Parameters();
+        }
+        expansionParameters.addParameter("url", new org.hl7.fhir.r4.model.UriType(url));
+        expansionParameters.addParameter("valueSetVersion", new org.hl7.fhir.r4.model.StringType(version));
```
Comment (ibaisi) [inline cqf-fhir-utility/src/main/java/org/opencds/cqf/fhir/utility/client/TerminologyServerClient.java:45]: maybe extract valueSetVersion to a const as it's used in many places?

**Review by barhodes — CHANGES_REQUESTED** [review]
My branch makes significant changes to this class.  Will need to discuss.

**Comment by TahaAttari** [comment]
superseded by !488

**Pull request closed without merging**